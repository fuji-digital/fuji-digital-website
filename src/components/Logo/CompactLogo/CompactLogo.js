import {
  StyledImageLogo,
  StyledTextLogo,
  StyledLogoContainer,
} from "./CompactLogo.styles.js";

import logo from "../../../images/logo/fuji-digital-logo.svg";
import textLogoWhite from "../../../images/logo/fuji-digital-text-logo-white.svg";

export const CompactLogo = () => {
  return (
    <StyledLogoContainer>
      <StyledImageLogo src={logo} alt="Fuji Digital Logo" />
      <StyledTextLogo
        data-testid="textLogo"
        src={textLogoWhite}
        alt="Fuji Digital Text Logo"
      />
    </StyledLogoContainer>
  );
};
