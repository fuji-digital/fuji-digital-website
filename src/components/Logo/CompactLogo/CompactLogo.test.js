import renderer from "react-test-renderer";
import { CompactLogo } from "./CompactLogo";

describe("CompactLogo icon", () => {
  it("matches snapshot", () => {
    const component = renderer.create(<CompactLogo />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
