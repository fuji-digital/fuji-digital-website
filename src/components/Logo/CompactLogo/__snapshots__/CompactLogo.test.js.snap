// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompactLogo icon matches snapshot 1`] = `
<div
  className="sc-eqUzNf etMiUw"
>
  <img
    alt="Fuji Digital Logo"
    className="sc-aYaIB kvsDPw"
    src="fuji-digital-logo.svg"
  />
  <img
    alt="Fuji Digital Text Logo"
    className="sc-gEvDqW eRcMlc"
    data-testid="textLogo"
    src="fuji-digital-text-logo-white.svg"
  />
</div>
`;

exports[`LinkedIn icon matches snapshot 1`] = `
<div
  className="sc-eqUzNf etMiUw"
>
  <img
    alt="Fuji Digital Logo"
    className="sc-aYaIB kvsDPw"
    src="fuji-digital-logo.svg"
  />
  <img
    alt="Fuji Digital Text Logo"
    className="sc-gEvDqW eRcMlc"
    data-testid="textLogo"
    src="fuji-digital-text-logo-white.svg"
  />
</div>
`;
